corehq/apps/reports/views.py

```diff
@@ -169,7 +168,9 @@
     get_group,
     group_filter,
     users_matching_filter,
-    resync_case_to_es)
+)
+from corehq.form_processor.utils.xform import resave_form as _resave_form
+from corehq.apps.hqcase.utils import resave_case as _resave_case
```

Comment:
Why have these been renamed to appear private?

Reply:
there are already methods with those names in the view. Is there a better way to avoid conflict?

Reply:
I think renaming the views would be better e.g. `resave_case_view`. That will also require changes to the urls file.

Reply:
Sounds good. updated.